unfoldingWord/translationCore
# Merge Conflict Check

#### This pull request addresses:
This PR brings in the Merge Conflict Check including the ability to find merge conflicts and resolve them in a usfm file. This PR also has small fixes to interface layout space to improve consistency, and actions for handling project import checks as a whole.

#### How to test this pull request:
Download these two projects:
[ema-x-ora_php_text_ulb.zip](https://github.com/unfoldingWord-dev/translationCore/files/1206115/ema-x-ora_php_text_ulb.zip)
[ha_tit_text_ulb.zip](https://github.com/unfoldingWord-dev/translationCore/files/1209066/ha_tit_text_ulb.zip)
Unzip them and open them with tC. 
You should be presented with the merge conflicts to choose. (Remember your choice of version to resolve)
Once you choose one to accept as the resolved git history, you can finish importing the project.
Check to make sure the version you chose shows in the check. (these projects have conflicts on the first check of tN to make it easier to verify.)
This PR assumes that merge conflicts are happening before the project is imported (a tC project or usfm file) and has a usfm file format for the conflict.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2392)
